pkg/background/update_request_controller.go

```diff
@@ -186,6 +180,9 @@ func (c *Controller) syncUpdateRequest(key string) error {
 		}
 		return fmt.Errorf("failed to fetch update request %s: %v", key, err)
 	}
+	if ur.Status.Handler != "" || ur.Status.State != kyvernov1beta1.Pending {
```

Comment:
Do we want to return if `ur.Status.State != kyvernov1beta1.Pending`?

How do we cleanup `completed` URs?

Reply:
Good question, shall i remove this ?

Reply:
Trying without...

Reply:
Removed.